### PR TITLE
Update 030420 1107

### PR DESCRIPTION
**Pull request by rugeGerritsen**
See changelog for changes.

manifest update: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2090

### CODE REVIEW

**Comment by rugeGerritsen** [comment]
@grochu , @MarekPieta , let me know if you want to do some manual testing

**Comment by pdunaj** [comment]
Hi @wiminordic , do you think it would be possible to trigger the same set of CI tests we execute for our PRs? If I recall correctly CI is looking for `nrf_desktop` in the title. Is it still true or has it changed?

**Comment by grochu** [comment]
> @grochu , @MarekPieta , let me know if you want to do some manual testing

Thanks. Good to go from the Protocols Team point of view.

File: mpsl/CHANGELOG.rst [hunk 1]
```diff
@@ -15,6 +15,12 @@ Added
 * Added API to set a TX Power envelope.
   Protocols using MPSL will limit their TX power to a value equal to, or lower than, the provided value.
 * Added support for using a low-swing and full-swing LF clock.
+* The support for integrating a 802.15.4 driver is now improved.
```
Comment (joerchan) [inline mpsl/CHANGELOG.rst:18]: should be "an 802.15.4" driver I think.

**Pull request merged**